src/meta/buying_options.yaml

```diff
@@ -1,18 +1,82 @@
-carrot:
+golden_carrot:
   type: item
   name: Golden Carrots
   item: minecraft:golden_carrot
   price: 200
   amount: 8
+torch:
+  type: item
+  name: Torches
+  item: minecraft:torch
+  price: 100
+  amount: 32
+hopper:
+  type: item
+  name: Hoppers
+  item: minecraft:hopper
+  price: 500
+  amount: 16
+cobblestone:
+  type: item
+  name: Cobblestone
+  item: minecraft:cobblestone
+  price: 10
+  amount: 64
+book:
+  type: item
+  name: Books
+  item: minecraft:book
+  price: 200
+  amount: 4
+firework:
+  type: item
+  name: Fireworks
+  item: minecraft:firework_rocket{Fireworks:{Flight:2}}
+  price: 100
+  amount: 32
+iron_pickaxe:
+  type: item
+  name: Iron Pickaxe
+  item: minecraft:iron_pickaxe
+  price: 500
+  amount: 1
+iron_shovel:
+  type: item
+  name: Iron Shovel
+  item: minecraft:iron_shovel
+  price: 400
+  amount: 1
+iron_axe:
+  type: item
+  name: Iron Axe
+  item: minecraft:iron_axe
+  price: 500
+  amount: 1
 night_vision:
   type: effect
   name: Night Vision
   item: minecraft:night_vision
   price: 250
   amount: 600
-torch:
-  type: item
-  name: Torches
-  item: minecraft:torch
-  price: 100
-  amount: 32
\ No newline at end of file
+fire_resistance:
+  type: effect
+  name: Fire Resistance
+  item: minecraft:fire_resistance
+  price: 500
+  amount: 600
+water_breathing:
+  type: effect
+  name: Water Breathing
+  item: minecraft:water_breathing
+  price: 250
+  amount: 600
+clear_weather:
+  type: command
+  name: Set Weather to clear
+  command: weather clear
+  price: 50
```

Comment:
I feel like this should either not exist (since it's something that affects everyone on the entire server), or it should be way more expensive.

Maybe something like 300?